tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+                         .format(tz_abbreviation, tz_name))
+
+            with allure.step('Get timezone details from timedatectl command'):
+                logging.info('Get timezone details from timedatectl command')
+                timedatectl_output = \
+                    ClockUtils.parse_linux_cmd_output(ClockUtils.run_cmd(duthosts, ClockConsts.CMD_TIMEDATECTL))
+                timedatectl_timezone = timedatectl_output[ClockConsts.TIME_ZONE]
+                timedatectl_tz_name = timedatectl_timezone.split()[0]
+                timedatectl_tz_abbreviation = timedatectl_timezone.split(' ')[1].split(',')[0].replace('(', '')
+                logging.info('Timezone in timedatectl: "{}"\nTimezone name: "{}"\nTimezone abbreviation: "{}"'
+                             .format(timedatectl_timezone, timedatectl_tz_name, timedatectl_tz_abbreviation))
+
+            with allure.step('Check that given timezone "{}" equals to timezone in timedatectl "{}"'
+                             .format(tz_name, timedatectl_tz_name)):
+                logging.info('Check that given timezone "{}" equals to timezone in timedatectl "{}"'
+                             .format(tz_name, timedatectl_tz_name))
+                ClockUtils.verify_value(expected=timedatectl_tz_name, actual=tz_name)
+
+            with allure.step('Check that given timezone abbreviation "{}" matches the expected timezone "{}"'
+                             .format(tz_abbreviation, tz_name)):
+                logging.info('Check that given timezone abbreviation "{}" matches the expected timezone "{}"'
+                             .format(tz_abbreviation, tz_name))
+                ClockUtils.verify_value(expected=timedatectl_tz_abbreviation, actual=tz_abbreviation)
+
+    @ staticmethod
+    def select_random_date():
+        """
+        @summary:
+            Select a random date
+        @return: a random date as string in the format "YYYY-MM-DD"
+        """
+        with allure.step('Select a random date'):
+            logging.info('Select a random date')
+            start_date = datetime.date.fromisoformat(ClockConsts.MIN_SYSTEM_DATE)
+            end_date = datetime.date.fromisoformat(ClockConsts.MAX_SYSTEM_DATE)
+
+            diff_days = (end_date - start_date).days
+
+            rand_num_of_days = random.randint(0, diff_days)
+
+            rand_date = start_date + datetime.timedelta(days=rand_num_of_days)
+
+            rand_date_str = rand_date.strftime('%Y-%m-%d')
+
+            logging.info('Selected random date: "{}"'.format(rand_date_str))
+            return rand_date_str
+
+    @ staticmethod
+    def select_random_time():
+        """
+        @summary:
+            Select a random time
+        @return: a random date as string in the format "hh:mm:ss"
+        """
+        with allure.step('Select a random time in a day'):
+            logging.info('Select a random time in a day')
+            rand_num_of_seconds_since_00 = random.randint(0, 24 * 60 * 60 - 1)
+
```

Comment:
Please remove extra space

Reply:
extra space where? i can't see it, sorry